lisa/features/hibernation.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.feature import Feature
+
+FEATURE_NAME_HIBERNATION = "Hibernation"
+
+
+class Hibernation(Feature):
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_HIBERNATION
+
+    @classmethod
+    def enabled(cls) -> bool:
+        return True
```

Comment:
Can we set it to false by default, I don't know how it affects the usage. But disable it by default should be better.